cookie-banner-rules-list.json

```
@@ -636,6 +636,265 @@
+      },
+      {
+        "name": "parental-control",
+        "value": "yes"
+      }
+    ]
+  },
+  "id": "02e04a3d-0848-4b10-a41d-80e31c4062a7"
+  },
+    {
+  "click": {
+    "optIn": ".btn--primary",
+    "presence": "div#cookie-consent"
+  },
+  "domain": "opera.com",
+  "schema": 1665669651587,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "cookie_consent_essential",
+        "value": "true"
+      },
+      {
+        "name": "cookie_consent_marketing",
+        "value": "true"
+      }
+    ]
+  },
+  "id": "09884f7b-ef06-45dd-9bda-960b5c6b7232"
+  },
+    {
+  "click": {
+    "optIn": "#cmpbntyestxt",
+    "optOut": "#cmpbntnotxt",
+    "presence": "div.cmpboxbtns"
+  },
+  "domain": "sourceforge.net",
+  "schema": 1665669888782,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "__cmpconsentx11319",
+        "value": "CPgx6wgPgx6wgAfUnBENCkCsAP_AAH_AACiQGgwDwAGgAVABAADQAJgAXwAwgB4AEEAJAAUYBCAFoALzAaCBoMA8ABoAFQAQAA0ACYAF8AMIAeABBACQAFGAQgBaAC8wGggAAA",
+        "isSecure": true
+      }
+    ],
+    "optOut": [
+      {
+        "name": "__cmpconsentx11319",
+        "value": "CPgx6wgPgx6wgAfUnBENCkCgAAAAAAAAACiQAAAAAAAA",
+        "isSecure": true
+      }
+    ]
+  },
+  "id": "786f7ae5-548a-43d4-97a2-7b8efb5ed7a7"
+  },
+    {
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "presence": "div#onetrust-button-group-parent"
+  },
+  "domain": "digicert.com",
+  "schema": 1665670173294,
+  "cookies": {},
+  "id": "0da6a0df-beac-4499-b3ac-023fc68ca695"
+  },
+    {
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "optOut": "button#onetrust-reject-all-handler",
+    "presence": "div#onetrust-button-group"
+  },
+  "domain": "webex.com",
+  "schema": 1665670505287,
+  "cookies": {},
+  "id": "5e777243-0819-48a6-90b4-66a1353e2514"
+  },
+    {
+  "click": {
+    "optIn": "div#gdpr-single-choice-overlay",
+    "presence": "button.wt-btn--filled"
+  },
+  "domain": "etsy.com",
+  "schema": 1665671188137,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "p",
+        "value": "eyJnZHByX3RwIjoxLCJnZHByX3AiOjF9"
+      }
+    ]
+  },
+  "id": "e5706488-f2e9-47bd-a178-4d569ca26ce8"
+  },
+    {
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "optOut": "button#onetrust-reject-all-handler",
+    "presence": "div#onetrust-button-group"
+  },
+  "domain": "indeed.com",
+  "schema": 1665671330451,
+  "cookies": {},
+  "id": "a4d94551-fc83-4477-b542-a13950b97468"
+  },
+    {
+  "click": {
+    "optIn": "button#L2AGLb",
+    "optOut": "button#W0wltc",
+    "presence": "div.spoKVd"
+  },
+  "domain": "google.co.in",
+  "schema": 1665671846324,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "NID",
+        "value": "511=LsN6Ln7-Mo3s1--Jj6XriDXGASnZN_hteNJEsV6PLicq77EDsKmJ8VTJZTDocymcQypv1LdFO5ybVOkZEZrNzxAn3T1F1OwY7lqNpB1LRkGd7LKLv_958U0Shm4Q2Pf4eqmSSZt54JGHrkp2bsttfFriwXdN1LkZzPG8DBxcbz0",
```

I think the trailing comma here is invalid and makes the JSON file invalid.
I was having problems parsing the file (until I ran it through Prettier, which oddly fixed the issue but didn't report the invalid JSON).

Might be worth investigating a linting script or something in CI to make sure the JSON can be parsed.
```suggestion
        "value": "511=LsN6Ln7-Mo3s1--Jj6XriDXGASnZN_hteNJEsV6PLicq77EDsKmJ8VTJZTDocymcQypv1LdFO5ybVOkZEZrNzxAn3T1F1OwY7lqNpB1LRkGd7LKLv_958U0Shm4Q2Pf4eqmSSZt54JGHrkp2bsttfFriwXdN1LkZzPG8DBxcbz0"
```

---

Removed the trailing comma